### DISCUSSION TITLE
Please consider releasing a native Windows ARM version

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#7115] [Ideas]
I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [X ] I've searched but have not found a similar issue.

**Describe the bug**
The app periodically crashes when run under the prism x86-arm interpreter.  There is a HUGE push to move from x86 to ARM by Qualcomm, Microsoft, Samsung, Lenovo, HP, and Dell for laptops.   While they are banking on Prism to fill the gaps, it is not perfect nor as fast as a native application.  Tuta has a big role to play in this success or failure story as it provides an essential service.  Moreover, it can provide this extended offering with minimal effort by creating the ARM pipeline on Git Hub.   Please consider helping us long-term customers out by supporting the more efficient ARM platform on Windows.  Furthermore, the new surface lineup is ONLY ARM, so not providing it is to deny customers the experience they purchased.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Desktop Environment, if applicable (on Linux)

**Additional context**

**Comment by ArcticEnigma** [comment]
I would also love to see this happen.
